src/index.ts

```diff
@@ -51,4 +52,5 @@ mongoose.connection.once('open', () => {
   // subscriberSafe();
   pullJobScheduler();
   customNotificationScheduler();
+  uploadSchemaFile();
```

Comment:
@estelafs 
don't rewrite the logic that already exists in src / utils / files / uploadFile.ts
all the logic to upload file is already there, so it's a duplication of code